wpilibc/athena/src/PIDController.cpp

```diff
@@ -332,6 +334,33 @@ void PIDController::SetOutputRange(double minimumOutput, double maximumOutput) {
 }
 
 /**
+ * Set the maximum error value that will cause the accumulated integral to be
+ * calculated. If the error term is large, it can potentially accumulate to have
+ * a huge effect on the output of the PID calculation. Any time the error is
+ * larger than the value specified in this function, the running integral will
+ * be set to zero causing it to not effect the output until the error is closer
+ * to the setpoint.
```

Comment:
This paragraph seems like it's slightly contradictory. Either we're simply not accumulating the integral outside of a set range, or we're *resetting* the integral outside of a set range. This paragraph seems to describe both, which is contradictory. 

Think of it as the difference between a "stop" and a "pause" button for the integral. We should reword this (and maybe rename the function) to be more clear about the actual algorithm we're trying to implement. The function name implies one, but the docs describe something different.

In fact, I can think of three different ways to approach this given the function name:

 - "pause" the integral, but keep the accumulated value, when outside of the I-band
 - reset the integral outside of the I-band
 - limit the amount added to the integral each term to the "max error to integrate"

We should make sure to differentiate between these three approaches in the function name so that the interface is clear and transparent.